app/vmauth/main.go

```diff
@@ -41,6 +41,8 @@ var (
 	reloadAuthKey        = flag.String("reloadAuthKey", "", "Auth key for /-/reload http endpoint. It must be passed as authKey=...")
 	logInvalidAuthTokens = flag.Bool("logInvalidAuthTokens", false, "Whether to log requests with invalid auth tokens. "+
 		`Such requests are always counted at vmauth_http_request_errors_total{reason="invalid_auth_token"} metric, which is exposed at /metrics page`)
+	brokenBeckendDeadlineTime = flag.Duration("brokenBackendDeadline", 3*time.Second, " When a particular backend in the URL list doesn't working, "+
```

Comment:
Current flag name seems to be confusing as it doesn't explain what it actually configures.
What do you think on naming it like `brokenBackendBackoff Configures interval during which backend will be skipped in load balancing after detecting backend failure.`?

Reply:
> What do you think on naming it like brokenBackendBackoff Configures interval during which backend will be skipped in load balancing after detecting backend failure.?

Yes it's right . but that flag name seems to be confising.., Could you suggest any other names?

Reply:
@zekker6 
Thanks! 
That's better than before! 
I push the new commit for that!